### DISCUSSION TITLE
Add option to open new tab in context menu

### DISCUSSION THREAD

**Discussion by eugbyte** [mbnuqw/sidebery#1455] [Ideas]
Add option to open new tab in context menu when right clicking on a tab. In Tree Style Tab for example,
![Screenshot (185)](https://github.com/mbnuqw/sidebery/assets/20870423/91afc348-fc3f-41ee-9959-a214b34544b5)

**Comment by QuaritschGerFfm** [comment]
Yes, that would be nice...a new tab on the same level as the selected tab. 😍 -> "New Tab below"

It is possible by pressing the middle mousebutton on the "+" to create a child tab 😍 ...that is great!
![grafik](https://github.com/user-attachments/assets/8ebbd48d-060b-4198-94fc-ad23094af086)

And default keybinding ctrl-shift-space also creates a tab below the selected, which is also great!
![grafik](https://github.com/user-attachments/assets/9a2bfda0-accb-483a-ba81-869ee98407f6)

But it would be nice, if it is also possible out of the context menu. All possibilities as shown in the above key-bindings. Configurable as all Tab-Menu entries. When you are using the mouse to select a Tab, where the new Tab should appear, than the context menu is the "nearest" way to do that (not to leave the hand from the mouse).
